plugins/experimental/uri_signing/jwt.c

```diff
@@ -112,13 +113,13 @@ jwt_validate(struct jwt *jwt)
     return false;
   }
 
-  if (!jwt->sub) { /* Mandatory claim. Will be validated after key verification. */
-    PluginDebug("Initial JWT Failure: missing sub");
+  if (!unsupported_string_claim(jwt->sub)) {
```

Comment:
The sub claim is not mandatory, even if included, so we probably should not be terminating just because it exists and is unsupported